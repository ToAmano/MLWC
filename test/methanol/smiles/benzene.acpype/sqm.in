Run semi-empirical minimization
 &qmmm
  qm_theory='AM1', grms_tol=0.05, scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   6     C        1.3820       -0.2140        0.0080 
   6    C1        0.5100       -1.3020       -0.0010 
   6    C2       -0.8690       -1.0910       -0.0100 
   6    C3       -1.3760        0.2090       -0.0060 
   6    C4       -0.5030        1.2970        0.0040 
   6    C5        0.8750        1.0860        0.0100 
   1     H        2.4560       -0.3780        0.0130 
   1    H1        0.9040       -2.3140       -0.0010 
   1    H2       -1.5480       -1.9390       -0.0190 
   1    H3       -2.4500        0.3730       -0.0120 
   1    H4       -0.8980        2.3100        0.0080 
   1    H5        1.5550        1.9340        0.0160 

