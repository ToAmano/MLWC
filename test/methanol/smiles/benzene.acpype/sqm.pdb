ATOM      1  C   UNL     1       1.382  -0.214   0.008  1.00  0.00           C
ATOM      2  C1  UNL     1       0.510  -1.302  -0.001  1.00  0.00           C
ATOM      3  C2  UNL     1      -0.869  -1.091  -0.009  1.00  0.00           C
ATOM      4  C3  UNL     1      -1.376   0.209  -0.006  1.00  0.00           C
ATOM      5  C4  UNL     1      -0.503   1.297   0.004  1.00  0.00           C
ATOM      6  C5  UNL     1       0.875   1.086   0.010  1.00  0.00           C
ATOM      7  H   UNL     1       2.469  -0.380   0.013  1.00  0.00           H
ATOM      8  H1  UNL     1       0.909  -2.327  -0.002  1.00  0.00           H
ATOM      9  H2  UNL     1      -1.557  -1.949  -0.018  1.00  0.00           H
ATOM     10  H3  UNL     1      -2.463   0.375  -0.012  1.00  0.00           H
ATOM     11  H4  UNL     1      -0.903   2.322   0.007  1.00  0.00           H
ATOM     12  H5  UNL     1       1.563   1.944   0.017  1.00  0.00           H
