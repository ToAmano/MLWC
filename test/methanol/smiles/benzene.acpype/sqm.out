            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 15

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       30
| QMMM:    diag iterations used for timing =       20
| QMMM:
| QMMM:              Internal diag routine = 0.019742 seconds
| QMMM:                 Dspev diag routine = 0.087195 seconds
| QMMM:                Dspevd diag routine = 0.059632 seconds
| QMMM:                Dspevx diag routine = 0.086785 seconds
| QMMM:                 Dsyev diag routine = 0.072225 seconds
| QMMM:                Dsyevd diag routine = 0.054452 seconds
| QMMM:                Dsyevr diag routine = 0.043382 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.012802 seconds
| QMMM:
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        1.3820   -0.2140    0.0080
  QMMM:     2        2      C        0.5100   -1.3020   -0.0010
  QMMM:     3        3      C       -0.8690   -1.0910   -0.0100
  QMMM:     4        4      C       -1.3760    0.2090   -0.0060
  QMMM:     5        5      C       -0.5030    1.2970    0.0040
  QMMM:     6        6      C        0.8750    1.0860    0.0100
  QMMM:     7        7      H        2.4560   -0.3780    0.0130
  QMMM:     8        8      H        0.9040   -2.3140   -0.0010
  QMMM:     9        9      H       -1.5480   -1.9390   -0.0190
  QMMM:    10       10      H       -2.4500    0.3730   -0.0120
  QMMM:    11       11      H       -0.8980    2.3100    0.0080
  QMMM:    12       12      H        1.5550    1.9340    0.0160

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -39.1466  -31.3701  -31.3700  -23.0603  -23.0601  -17.8556  -16.1225  -15.4003
 -14.1608  -14.1607  -13.3813  -11.8881  -11.8880   -9.6529   -9.6529    0.5547
   0.5548    2.9778    4.0359    4.0360    4.0476    4.1928    4.5985    4.5986
   5.1229    5.1230    5.6119    5.6120    5.7290    6.1333

 Heat of formation   =         21.93339196 kcal/mol  (        0.95110325 eV)

 Total SCF energy    =     -19609.69477893 kcal/mol  (     -850.34017514 eV)
 Electronic energy   =     -75026.94948308 kcal/mol  (    -3253.41266567 eV)
 Core-core repulsion =      55417.25470415 kcal/mol  (     2403.07249053 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.130
     2      C                 -0.130
     3      C                 -0.130
     4      C                 -0.130
     5      C                 -0.130
     6      C                 -0.130
     7      H                  0.130
     8      H                  0.130
     9      H                  0.130
    10      H                  0.130
    11      H                  0.130
    12      H                  0.130
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE     0.000   -0.000   -0.000    0.000
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        1.3821   -0.2136    0.0078
  QMMM:     2        2      C        0.5098   -1.3022   -0.0013
  QMMM:     3        3      C       -0.8692   -1.0910   -0.0094
  QMMM:     4        4      C       -1.3758    0.2087   -0.0063
  QMMM:     5        5      C       -0.5034    1.2974    0.0040
  QMMM:     6        6      C        0.8754    1.0862    0.0102
  QMMM:     7        7      H        2.4691   -0.3799    0.0132
  QMMM:     8        8      H        0.9090   -2.3269   -0.0020
  QMMM:     9        9      H       -1.5566   -1.9493   -0.0183
  QMMM:    10       10      H       -2.4627    0.3753   -0.0120
  QMMM:    11       11      H       -0.9029    2.3219    0.0073
  QMMM:    12       12      H        1.5632    1.9443    0.0166

           --------- Calculation Completed ----------

